src/revChatGPT/revChatGPT.py

```diff
@@ -374,6 +374,51 @@ def login(self, email: str, password: str) -> None:
             raise Exception("Error logging in")
 
 
+    def send_feedback(
+        self,
+        is_good: bool,
+        is_harmful=False,
+        is_not_true=False,
+        is_not_helpful=False,
+        description=None,
+    ):
+        from enum import Enum
+
+        class ChatGPTTags(Enum):
+            Harmful = "harmful"
+            NotTrue = "false"
+            NotHelpful = "not-helpful"
```

Comment:
@alexandreteles Mentioned Enum as an alternative. However, since dataclasses are supported much earlier, I recommend using those instead.

Reply:
@acheong08 got it.